### PR TITLE
DJI Test 1-2 Login/Loguot

### PR DESCRIPTION
**Pull request by YevheniiBond**

### CODE REVIEW

**Review by Safron1ta2 — APPROVED** [review]

File: src/test/java/YevheniiBondarenko/Tests/Tests/DJITest1E2EbuyingCamera.java [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+import BasePageAndTestInit.TestInit;
+import YevheniiBondarenko.Tests.Pages.DJIWebElement;
+import org.openqa.selenium.interactions.Actions;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+public class DJITest1E2EbuyingCamera extends TestInit {
+
+    @Test
+
```
Comment (Wolmeright) [inline src/test/java/YevheniiBondarenko/Tests/Tests/DJITest1E2EbuyingCamera.java:12]: Delete empty line, please

File: src/test/java/YevheniiBondarenko/Tests/Tests/DJITest2LogInLogOut.java [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+    public void djiTest2LogInLogOut() {
+        DJIWebElement djiWebElement = new DJIWebElement(driver);
+        Actions action = new Actions(driver);
+
+        openUrl("https://dji-kyiv.com/");
+        djiWebElement.enter().click();
+        djiWebElement.inputEmail().sendKeys("interdancecoach@gmail.com");
+        djiWebElement.inputPassword().sendKeys("1234567888\n");
+
+        Assert.assertTrue(djiWebElement.userName().isDisplayed());
```
Comment (Wolmeright) [inline src/test/java/YevheniiBondarenko/Tests/Tests/DJITest2LogInLogOut.java:20]: The test falls on 20 line

**Review by Wolmeright — CHANGES_REQUESTED** [review]

File: src/test/java/YevheniiBondarenko/Tests/Tests/DJITest2LogInLogOut.java (hunk 2 above)
[the remark this answers is not in the document]
Comment (YevheniiBond) [reply src/test/java/YevheniiBondarenko/Tests/Tests/DJITest2LogInLogOut.java:21]: Done

**Review by Tsibatyuk — APPROVED** [review]

File: src/test/java/YevheniiBondarenko/Tests/Tests/DJITest1E2EbuyingCamera.java (hunk 1 above)
Comment (YevheniiBond) [reply src/test/java/YevheniiBondarenko/Tests/Tests/DJITest1E2EbuyingCamera.java:12]: I don't think so)

Comment (AndriiMyronenko418) [reply src/test/java/YevheniiBondarenko/Tests/Tests/DJITest1E2EbuyingCamera.java:12]: we don't use empty lines after @Test

File: src/test/java/YevheniiBondarenko/Tests/Tests/DJITest1E2EbuyingCamera.java [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+    public void djiTestOne(){
+        DJIWebElement djiWebElement = new DJIWebElement(driver);
```
Comment (AndriiMyronenko418) [inline src/test/java/YevheniiBondarenko/Tests/Tests/DJITest1E2EbuyingCamera.java:14]: please add empty line between 13 and 14 line

File: src/test/java/YevheniiBondarenko/Tests/Tests/DJITest2LogInLogOut.java [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+import org.openqa.selenium.interactions.Actions;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+import static org.openqa.selenium.Keys.ENTER;
+
+public class DJITest2LogInLogOut extends TestInit {
+    @Test
+    public void djiTest2LogInLogOut() {
+        DJIWebElement djiWebElement = new DJIWebElement(driver);
+        Actions action = new Actions(driver);
```
Comment (AndriiMyronenko418) [inline src/test/java/YevheniiBondarenko/Tests/Tests/DJITest2LogInLogOut.java:14]: please add empty lines:
1. between 10 and 11
2. between 12 and13

File: src/test/java/YevheniiBondarenko/Tests/Tests/DJITest1E2EbuyingCamera.java (hunk 1 above)
Comment (YevheniiBond) [reply src/test/java/YevheniiBondarenko/Tests/Tests/DJITest1E2EbuyingCamera.java:12]: Ok

File: src/test/java/YevheniiBondarenko/Tests/Tests/DJITest1E2EbuyingCamera.java (hunk 3 above)
Comment (YevheniiBond) [reply src/test/java/YevheniiBondarenko/Tests/Tests/DJITest1E2EbuyingCamera.java:14]: +

File: src/test/java/YevheniiBondarenko/Tests/Pages/DJIWebElement.java [hunk 5]
```diff
@@ -0,0 +1,74 @@
+package YevheniiBondarenko.Tests.Pages;
+
+import BasePageAndTestInit.BasePage;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+
+import java.util.List;
+
+public class DJIWebElement extends BasePage {
+    public DJIWebElement(WebDriver driver) {
```
[the remark this answers is not in the document]
Comment (YevheniiBond) [reply src/test/java/YevheniiBondarenko/Tests/Pages/DJIWebElement.java:9]: Done

**Pull request closed without merging**